packages/react/src/components/accordion/index.tsx

```diff
@@ -0,0 +1,40 @@
+import * as RadixAccordion from "@radix-ui/react-accordion";
+import { clsx } from "clsx";
+import { ReactNode } from "react";
+import { ChevronLeft } from "../icon/chevron-left";
+import * as styles from "./style.css";
+
+type Item = {
+  value: string;
+  icon?: ReactNode;
+  trigger: ReactNode;
+  content: ReactNode;
+};
+
+type Props = {
+  items: Item[];
+  className?: string;
+  itemClassName?: string;
+};
+
+export const Accordion = ({ items, className, itemClassName }: Props) => (
```

Comment:
@ahmed-gz this will be for internal use but still it would be useful to have some docs here on how to use it